image/image.go

```diff
@@ -402,6 +411,10 @@ func bootstrapToRootDir(tsto *ToolingStore, model *asserts.Model, opts *Options,
 	if len(locals) > 0 {
 		fmt.Fprintf(Stderr, "WARNING: %s were installed from local snaps disconnected from a store and cannot be refreshed subsequently!\n", strutil.Quoted(locals))
 	}
+	// XXX: make this a warning only?
+	if model.Base() != "" && !seen["snapd"] {
+		return fmt.Errorf(`Base %q is used but no "snapd" snap is installed`, model.Base())
```

Comment:
forcing people to add snapd themselves seems strange, see same argument on the forum